### PR TITLE
Correctly terminate the process after interrupt

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
This stops the pager from breaking the terminal

### CODE REVIEW

File: bin/job [hunk 1]
```diff
@@ -59,8 +59,9 @@ begin
   end
 rescue Interrupt
   if Kernel.const_defined?(:Paint)
-    $stderr.puts "\n#{Paint['WARNING', :underline, :yellow]}: Cancelled by user"
+    $stderr.print "\n#{Paint['WARNING', :underline, :yellow]}: Cancelled by user"
```
Comment (benarmston) [inline bin/job:62]: Why the change from `puts` to `print`?

Comment (WilliamMcCumstie) [reply bin/job:62]: > See for full details:
> https://trello.com/c/9sWZKxne/282-ttypager-breaks-the-terminal-on-interrupt#comment-60884aa45fc8de123ca1e421

tl;dr The `raise SignalException.new(2)` causes an additional line to be emitted. So switching to `print` accounts for it.

**Comment by benarmston** [comment]
This is definitely better than it was before.  I'm getting behaviour where the first keypress after is ignored, but the terminal doesn't need resetting.  I'm happy to merge this once my question above is answered.

File: bin/job (hunk 1 above)
Comment (WilliamMcCumstie) [reply bin/job:62]:
```
$ irb
irb(main):001:0> raise SignalException.new(2)

$
```
